### DISCUSSION TITLE
edit rate limiting, using docker version

### DISCUSSION THREAD

**Discussion by arielesp** [chatwoot/chatwoot#5784] [Self hosted] [answered]
Hi
How could I change rate limiting of chatwoot, when using docker version?

thanks in advance.

source: https://www.chatwoot.com/docs/self-hosted/monitoring/rate-limiting/

**Comment by vishnu-narayanan** [comment, marked as answer]
If you want to modify the limits, you should edit [this file](https://github.com/chatwoot/chatwoot/blob/develop/config/initializers/rack_attack.rb) to suit your needs and rebuild the docker image.

To disable the `rack_attack` gem, set  `ENABLE_RACK_ATTACK` env variable to `false`.
